gribapi/gribapi.py

```diff
@@ -2060,6 +2060,9 @@ def grib_get_api_version():
     def div(v, d):
         return (v / d, v % d)
 
+    if not lib:
+        raise Exception("Could not load the ecCodes library!")
```

Comment:
@shahramn it is not custom to raise from the raw `Exception` class, that would best be a RuntimeError exception. Can you update the the PR?

Reply:
Do you mean use "raise RuntimeError" instead?

Reply:
OK I updated the develop and pushed